pkg/controller/azdrivernode_controller.go

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
Including `controller` in the file name is redundant. This should just be `azdrivernode.go`

Reply:
`// +build azurediskv2`